### DISCUSSION TITLE
Urlopen error [Errno 113] No route to host

### DISCUSSION THREAD

**Discussion by jorpyy** [flet-dev/flet#4257] [Q&A] [answered]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

When I try to run the app on the Flet app  I get this error:
`Error connecting to http://192.168.0.106:8551/english_alphabet_app/main.py: <urlopen error [Errno 113] No route to host>`
My android device and computer are connected to the same Wi-Fi.

### Code sample

<details open><summary>Code</summary>

```python
import flet as ft

def main(page: ft.Page):
    page.add(ft.SafeArea(ft.Text("Hello, Flet!")))

ft.app(main)
```

</details>

### To reproduce

Run the code on the Flet app and see the error.

### Screenshots / Videos

<details open>
<summary>Captures</summary>

![Flet](https://github.com/user-attachments/assets/b11c0fb3-74a2-4c6b-b24c-59d368ba999f)

</details>

### Operating System

Linux

### Operating system details

EndeavourOS

### Flet version

0.24.1

### Regression

No, it isn't

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

**Comment by FeodorFitsner** [comment]
Try opening that URL in the browser on the same mobile phone?

**Reply by jorpyy** [reply]
![unreachable](https://github.com/user-attachments/assets/2676b2d3-58b7-4eaa-a77b-f8668b2752b6)

**Reply by FeodorFitsner** [reply, marked as answer]
Try using hostname instead of IP, make sure 8551 port is opened on Firewall (what OS is on your computer?).

**Reply by jorpyy** [reply]
EndeavourOS

**Reply by FeodorFitsner** [reply]
Well, I'm afraid I don't have any experience with EndeavourOS. Anyway, you should be able to open that URL in the phone browser to make it work in Flet app.

**Reply by jorpyy** [reply]
> Try using hostname instead of IP, make sure 8551 port is opened on Firewall (what OS is on your computer?).

After adding the port on Firewall it worked.
